tests/integration/conftest.py

```diff
@@ -163,7 +165,35 @@ def protostar_fixture(
     session_mocker: MockerFixture,
     protostar_project_root_path: Path,
 ) -> ProtostarFixture:
+    """@deprecated: Use `create_protostar_project` fixture instead."""
     return build_protostar_fixture(
         mocker=session_mocker,
         project_root_path=protostar_project_root_path,
     )
+
+
+class CreateProtostarProjectFixture(Protocol):
+    def __call__(self, project_root_path: Path) -> ContextManager[ProtostarFixture]:
+        ...
+
+
+ProtostarProjectFixture = ProtostarFixture
+
+
+@pytest.fixture(name="create_protostar_project", scope="module")
+def create_protostar_project_fixture(
+    session_mocker: MockerFixture,
+):
+    @contextmanager
+    def create_protostar_project(project_root_path: Path):
```

Comment:
This function returns protostar fixture instance, but you're naming the variables `protostar_project`, this seems misleading to what this really returns

Reply:
Removed the concept of `ProtostarProject` but left the name of this function, because `create_protostar` would be more misleading.